### DISCUSSION TITLE
Not applying tailwindcss classes to daisyui classes

### DISCUSSION THREAD

**Discussion by Arman-Espiar** [saadeghi/daisyui#3210] [Q&A]
Tailwindcss classes don't work on daisyui classes when using daisyui (specifically I made my own theme by theme maker)!

 For example  `<div class="flex w-full justify-center items-center input input-secondary outline-none focus:outline-none"> `section outline-none is not applied.

I want to know if my statement is correct or is there a problem because I gave these classes to the div?

**Comment by saadeghi** [comment]
Please provide a reproduction link where I can see the issue

**Reply by Arman-Espiar** [reply]
https://play.tailwindcss.com/NVOZr2Q5L4 ,In this example, the `outline-none focus:outline-none section` does not work, also in the `svg` section, the stroke property value, for example: `stroke="stroke-rose-600"` does not work correctly except with the `stroke-current` value.I may be wrong, it is better to test these codes locally.
